### ISSUE TITLE
Couldn't run generate_tfrecord.py

### ISSUE THREAD

**Issue by AniTho** [EdjeElectronics/TensorFlow-Object-Detection-API-Tutorial-Train-Multiple-Objects-Windows-10#386]
While running the generate_tfrecord.py, encountering this error.
![Screenshot (14)](https://user-images.githubusercontent.com/34787227/67971842-88ca4600-fc33-11e9-8afb-663005d6690a.png)

**Comment by Nightrain** [comment]
In your generate_tfrecord.py file make sure the last else statement is return 0 instead of None

`def class_text_to_int(row_label):
    if row_label == 'basketball':
        return 1
    elif row_label == 'shirt':
        return 2
    elif row_label == 'shoe':
        return 3
    else:
        None`

**Comment by Nightrain** [comment]
I tried to use the markup for adding code, sorry it did not stay formatted. but you should figure it out.

**Comment by nika9774** [comment]
did you figured out how to solve the problem? because i have the same error

**Comment by Nightrain** [comment]
@nika9774 
Yes I did.  It is like in the post above.  In the generate_tfrecord.py look for the function you modified to match your lablemap.pbtxt file.  At the end of the IF statement is the else:None to wrap it up.  You have to change that else:None to else:0. (else:zero) That function (class_text_to_int) needs to return all ints, it is the None being returned that is causing the problem downstream.

**Comment by nika9774** [comment]
thank you. it worked!

**Comment by Nightrain** [comment]
Good to hear it have fun

**Comment by hoanglvdev** [comment]
![image](https://user-images.githubusercontent.com/73802582/113504421-3268e180-9562-11eb-9bdd-e5a5675e85c4.png)
i have problem when i run file generate_tfrecord.py.
